tempesta_fw/t/unit/test_http_parser.c

```diff
@@ -47,11 +49,88 @@ reset_req(size_t data_len)
 	alloc_req(data_len);
 }
 
+static void
+alloc_resp(size_t data_len)
+{
+	BUG_ON(resp);
+	resp = test_resp_alloc(data_len);
+}
+
+static void
+free_resp(void)
+{
+	if (resp)
+		test_resp_free(resp);
+	resp = NULL;
+}
+
+static void
+reset_resp(size_t data_len)
+{
+	free_resp();
+	alloc_resp(data_len);
+}
+
+static void
+free_msgs(void)
+{
+	free_req();
+	free_resp();
+}
+
+int
+split_and_parse_n(unsigned char *str, int type, size_t len, int chunks)
+{
+	int chlen = len / chunks, rem = len % chunks, pos = 0, step, r;
+	char tmp1, tmp2;
+
+	while (pos < len)
+	{
```

Comment:
This doesn't satisfy our coding style: https://github.com/krizhanovsky/linux-3.10.10-sync_sockets/blob/master/Documentation/CodingStyle